crates/meilisearch/src/search/mod.rs

```diff
@@ -648,15 +663,17 @@ pub struct SearchHit {
     pub ranking_score_details: Option<serde_json::Map<String, serde_json::Value>>,
 }
 
-#[derive(Serialize, Clone, PartialEq)]
+#[derive(Serialize, Clone, PartialEq, ToSchema)]
 #[serde(rename_all = "camelCase")]
+#[schema(rename_all = "camelCase")]
 pub struct SearchResult {
     pub hits: Vec<SearchHit>,
     pub query: String,
     pub processing_time_ms: u128,
     #[serde(flatten)]
     pub hits_info: HitsInfo,
     #[serde(skip_serializing_if = "Option::is_none")]
+    #[schema(value_type = HashMap<String, Value>)]
```

Comment:
small question, but other types have BTreeMap and this one uses HashMap, is there any specific reason why?

Also should we put the option here, or is the "optionality" of the field already handled by the actual type being an option?